docs/modules/kafka_ingester.md

```diff
@@ -16,21 +16,34 @@ logKafkaEvents: false
 logKafkaErrors: true
 # List of Kafka brokers
 brokers:
-  - <string>
+  - <string> # e.g. kafka-1.example.com:9092
 topic: ""
 groupId: ""
+# commitInterval indicates the interval at which offsets are committed to the broker.
+# If 0 (default), commits will be handled synchronously.
+commitInterval: <duration> # e.g. 0, 5s, 10m
+# retentionTime optionally sets the length of time the consumer group will be saved by the broker.
+# Default: 24h
+retentionTime: <duration>
+# lastOffsetAsDefault determines from whence the consumer group should begin consuming when it finds a partition without a committed offset.
+# If true, partition will be read from its LastOffset, otherwise from the FirstOffset.
+lastOffsetAsDefault: false
```

Comment:
Suggest `kafkaStartOffset` with possible values `lastOffset` and `FirstOffset`. This naming will be also straightforward for kafka "experts".

Reply:
https://github.com/seznam/slo-exporter/pull/48#discussion_r599936537